CODE_OF_CONDUCT.md

```diff
@@ -1,76 +1,120 @@
+
+If the situation is not resolved within one week, we will respond within one week to the original reporter with an update and explanation. Once we have determined our response, we will separately contact the original reporter and other individuals to let them know what actions (if any) will be taken. We will take into account feedback from the individuals involved on the appropriateness of our response, but we dont guarantee well act on it.
+
+## Ombudsperson
+
+The SXS ombudsperson is a senior member of the collaboration, either faculty or a senior researcher, and may be a member of the executive committee. The executive committee will solicit volunteers for the position. The executive committee will choose the ombudsperson by holding an election with ranked-choice voting among those nominees that agreed to stand for election. The ombudsperson serves a two-year term.
+
+The Ombudsperson (currently Geoffrey Lovelace, email ombudsperson@black-holes.org) offers confidential, neutral, and informal conflict resolution services to members of the SXS Collaboration. All collaboration members should feel free to contact the Ombudsperson to informally and confidentially discuss any concerns that they might have regarding conflicts, problems, violations of the code of conduct, or any other concerns they might have.
+
+After speaking with Ombudsperson, if you wish, you can choose to ask the Ombudsperson to bring your concerns to the attention of the SXS Executive Committee or to other people at the appropriate institutions, and you can also choose to report your concerns formally to the SXS Executive Committee (as outlined above). Otherwise, unless there is a serious issue of safety, your conversations with the Ombudsperson will remain confidential.
+
+We have adopted the same policy as the LIGO Scientific Collaboration Ombudsperson; see this document, replacing LSC with SXS Collaboration, and LSC Spokesperson with the SXS Executive Committee, for more details about the [Ombudspersons role](https://dcc.ligo.org/public/0099/M1300006/001/LSCOmbudsperson.pdf).
+
+
+## Advocate
+
+The SXS Advocate must be elected by the student and postdoc members of SXS, and be within two years of their Ph.D. (i.e. a senior graduate student or a new postdoc). The SXS executive committee will have a call for nominations and then consult with nominees to see if they are willing to be the advocate. If the SXS executive committee has any concerns with a nominee, they will discuss these concerns with the nominator. The SXS executive committee will then supervise an election among SXS student and postdoc members with ranked-choice voting among those nominees that agreed to stand for election. Once the list of candidates for SXS advocate is announced, the executive committee cannot change the candidate list (unless there is a serious violation of the SXS code of conduct). The Advocate is elected for a one-year term, and may be re-elected.
```

Comment:
These two sections seem to duplicate the content of (new) lines 23-31 above

Reply:
This is directly copied from the current code of conduct, so to change this duplication we'll have to all discuss and adopt it

Reply:
I actually don't remember where our definitive CoC lives. I looked at [this one](https://github.com/sxs-collaboration/WelcomeToSXS/blob/master/sxs_code_of_conduct.pdf) which agrees with what's on the black-holes.org web site, which must be older, and don't have this duplication.

So, separate issues are:
- Fix the duplication at the origin (where?)
- Update the web site and Welcome repo (linked above) with the new version